FrenchFilter/sections/whitelist.txt

```diff
@@ -6,6 +6,8 @@
 /^(https?|wss?):\/\/([0-9a-z\._-]+)\.(accountant|bid|cf|click|club|com|cricket|date|download|faith|fun|ga|gdn|gq|info|life|link|loan|men|ml|network|online|ovh|party|pro|pw|racing|review|rocks|ru|science|site|space|stream|tk|top|trade|uk|webcam|website|win|xyz|zone)\.\/(.*)/$document,image,popup,script,stylesheet,subdocument,websocket,xmlhttprequest,badfilter
 ! https://github.com/AdguardTeam/AdguardFilters/issues/61786
 willhaben.at#@#.adHeading
+! https://github.com/AdguardTeam/AdguardFilters/issues/61823
+@@||voiranime.com/wp-content/cache/minify/7a103.js
```

Comment:
also  not blocked

Reply:
For fixing switching between players
![Screenshot 2020-08-21 at 11 23 46](https://user-images.githubusercontent.com/18536328/90869492-f4d39180-e3a0-11ea-92a6-f3430a40a203.png)